code/game/turfs/simulated.dm

```diff
@@ -17,7 +17,7 @@
 	if(wet >= wet_setting)
 		return
 	wet = wet_setting
-	if(wet_setting == TURF_WET_WATER)
+	if(wet_setting == TURF_WET_WATER || TURF_WET_LUBE)

```

Comment:
but it literally does work
i just compiled it and tested it ingame
???

i guess i could fix it to what you guys are suggesting anyways since those should work too, but i don't really see a point and i REALLY don't want to delve back into git hell again :cry: